lm_eval/models/optimum_lm.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+from lm_eval.api.registry import register_model
+from lm_eval.models.huggingface import HFLM
+
+
+@register_model("optimum-causal")
+class OptimumLM(HFLM):
+    """
+    Optimum Intel provides a simple interface to optimize Transformer models and convert them to \
+    OpenVINO™ Intermediate Representation (IR) format to accelerate end-to-end pipelines on \
+    Intel® architectures using OpenVINO™ runtime.
+    """
+
+    def __init__(
+        self,
+        device = "cpu",
+        **kwargs,
+    ) -> None:
+        
+        if "backend" in kwargs:
+            # optimum currently only supports causal models.
+            assert kwargs["backend"] == "causal"
+        else:
+            raise Exception("Please be sure your model is a `causal` model.")
+
+        assert device == "cpu"
+        super().__init__(device=device, **kwargs)
+
+    def _create_model(
+        self,
+        pretrained: str,
+        revision = "main",
+        dtype = "auto",
+        trust_remote_code = False,
+        **kwargs,
+    ) -> None:
+        
+        try:
+            import optimum    
+        except ModuleNotFoundError: 
+            raise Exception("package `optimum` is not installed. Please install it via `pip install optimum[openvino] ipywidgets pillow torchaudio`")
+        from optimum.intel.openvino import OVModelForCausalLM
+
+        model_kwargs = kwargs if kwargs else {}
+        if Path("/pretrained/openvino_model.xml").exists():
+            export = False
+        else:
+            export = True
```

Comment:
What's the intent of this export to a hardcoded path? Do other parts of the evaluation harness re-use this on-disk exported model?